modules/user/user.repository.js

```diff
@@ -57,8 +57,31 @@ class UserRepository {
         return userDB;
     }
 
-    async readAllUsers() {
-        const usersArr = await UserModel.find({}, userAggregateForm);
+    async readAllUsers(searchParams, sortParams, limit, skip) {
```

Comment:
In this case we have only search params, but in general search = filtering, so here could be other filtering criteria's so this trick with 
 `     Object.entries(searchParams).map(function ([key, value]) {
            searchParamsMongo[key] = new RegExp(value, "gi");
        });`
will not work.
Don't need any changing, just keep in mind that here could be some other filtering besides searching.